types/umzug/index.d.ts

```diff
@@ -213,6 +213,11 @@ declare namespace umzug {
     interface Migration {
```

Comment:
I'm not familiar with umzug, but do you need [options](https://github.com/sequelize/umzug/blob/4e3169dd9d6051af46987a97836ed47aa013ebb2/src/migration.js#L37) as well?

Reply:
Actually not and I also just removed the other two props (`file` and `path`) from the interface. Migration is the interface and https://github.com/sequelize/umzug/blob/master/src/migration.js is just one specific implementation of the same. `file`, `path` and `options` are only used in the internal implementation but if you want to pass a custom migration (via my originally suggested changes) then you just need to implement the functions.

Reply:
Cool. Thanks!